app/controllers/articles_controller.rb

```diff
@@ -35,4 +36,14 @@ def set_categories
       active_category category.parent_id if category.child?
     end
   end
+
+  def set_show_newsletter_signup
+    exclusions = YAML.load(
+      File.read(File.expand_path("#{Rails.root}/config/newsletter_article_exclusion_list.yml", __FILE__))
+    )
+
+    @newsletter_excluded = exclusions.reject! {
```

Comment:
Avoid using {...} for multi-line blocks.<br>Avoid multi-line ?: (the ternary operator); use `if`/`unless` instead.